Buy/Client/Graph.Client.swift

```diff
@@ -82,9 +82,9 @@ extension Graph {
         ///     - locale:     The buyer's current locale. Supported values are limited to locales available to your shop.
         ///
         public init(shopDomain: String, apiKey: String, session: URLSession = URLSession(configuration: URLSessionConfiguration.default), locale: Locale? = nil) {
-
+            
             let shopURL  = Client.urlFor(shopDomain)
-            self.apiURL  = Client.urlFor(shopDomain, path: "/api/2020-01/graphql")
+            self.apiURL  = Client.urlFor(shopDomain, path: String(format:"/api/%@/graphql", Storefront.Schema.version))
```

Comment:
The Swift convention is not using `String(format:)`, that's actually an Objective-C function call. With Swift we can composite strings inline:
```swift
Client.urlFor(shopDomain, path: "/api/\(Storefront.Schema.version)/graphql")
```